.npmignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
+
+node_modules/
+out/
```

Comment:
We actually want to npmignore `src` not `out`: the npm package should contain the compiled code rather than the typescript sources